irmaclient/session.go

```diff
@@ -255,27 +246,20 @@ func (session *session) getSessionInfo() {
 	session.processSessionInfo()
 }
 
-func serverName(hostname string, request irma.SessionRequest, conf *irma.Configuration) irma.TranslatedString {
-	sn := irma.NewTranslatedString(&hostname)
-
-	if ir, ok := request.(*irma.IssuanceRequest); ok {
-		// If there is only one issuer in the current request, use its name as ServerName
-		var iss irma.TranslatedString
-		for _, credreq := range ir.Credentials {
-			credIssuer := conf.Issuers[credreq.CredentialTypeID.IssuerIdentifier()].Name
-			if !reflect.DeepEqual(credIssuer, iss) { // Can't just test pointer equality: credIssuer != iss
-				if len(iss) != 0 {
-					return sn
-				}
-				iss = credIssuer
-			}
-		}
-		if len(iss) != 0 {
-			return iss
-		}
+func requestorInfo(serverURL string, request irma.SessionRequest, conf *irma.Configuration) *irma.RequestorInfo {
+	if serverURL == "" {
+		return nil
 	}
+	u, _ := url.ParseRequestURI(serverURL) // Qr validator already checked this for errors
+	hostname := u.Hostname()
+	info, present := conf.Requestors[hostname]
 
-	return sn
+	if (u.Scheme == "https" || !common.ForceHTTPS) && present &&
```

Comment:
`ForceHTTPS` is true by default except for in tests. This means that `http://` requestors in the requestor schemes are never replaced with the name, also not when developer mode is enabled in the IRMA app. Was this the intension?

Reply:
Yes, as SSL provides the needed assurance on identity to make it reasonable to use a pretty verifier name.